chore(http): Refresh some headers pages (~a-c) (#36038)
mdn/content: files/en-us/web/http/headers/content-language/index.md

> Do **not** use this meta element like this for stating a document language:

Reviewer: ```suggestion
Do **not** use this meta element to state the document language, as shown below:
```

```html example-bad

> <!-- This is bad practice! -->

Reviewer: This is not needed because we've already said it above the code snippet and the code is also getting highlighted as a bad example
```suggestion
```
